addons/website_slides_survey/models/survey_survey.py

```diff
@@ -41,7 +41,8 @@ def _unlink_except_linked_to_course(self):
                 for certi in certifications
             ]
             raise ValidationError(_(
-                'Any Survey listed below is currently used as a Course Certification and cannot be deleted:\n%s',
+                'Uh-oh! You can’t delete a survey used as a Course Certification! Otherwise, students might think diplomas just grow on trees.\n'
+                'The courses that need it are:\n%s',
```

Comment:
```suggestion
                'Uh-oh! You can’t delete surveys used as a Course Certification! Otherwise, students might think diplomas just grow on trees.\n'
                'The courses that need them are:\n%s',
```